### PR TITLE
Slight improvement for docs of std.array.join

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

File: std/array.d [hunk 1]
```diff
@@ -1542,18 +1542,18 @@ unittest
...
 /++
-   Concatenates all of the ranges in $(D ror) together into one array using
-   $(D sep) as the separator if present.
+   Eagerly concatenates all of the ranges in `ror` together (with the GC)
+   into one array using `sep` as the separator if present.
 
    Params:
-        ror = Range of Ranges of Elements
-        sep = Range of Elements
+        ror = A range of input ranges of elements
```
Comment (qznc) [inline std/array.d]: It's an "input range of input ranges of elements" then?

**Comment by JackStouffer** [comment]
@qznc fixed

**Comment by qznc** [comment]
LGTM

**Comment by Hackerpilot** [comment]
Auto-merge toggled on

**Pull request merged**